cmd/soci/commands/create.go

```diff
@@ -55,6 +56,10 @@ var CreateCommand = cli.Command{
 			Usage: "Minimum layer size to build zTOC for. Smaller layers won't have zTOC and not lazy pulled. Default is 10 MiB.",
 			Value: 10 << 20,
 		},
+		cli.BoolTFlag{
+			Name:  disableXAttrsFlag,
+			Usage: "Specify this flag to skip adding DisableXAttrs annotation for every layer.",
```

Comment:
We need to update this to specify it adds the disable xattrs optimization and default value of true. To disable, users will need to `--disable-xattrs=false`.